mlos_bench/mlos_bench/config/schemas/services/remote/azure/azure-auth-service-subschema.json

```diff
@@ -23,6 +23,18 @@
                         "tokenRequestInterval": {
                             "description": "Time before token expiration to request a new token",
                             "type": "number"
+                        },
+                        "keyVaultName": {
+                            "description": "Name of key vault to retrieve the certificate from, for authorizing the Service Principal",
+                            "type": "string"
+                        },
+                        "certName": {
+                            "description": "Name of the Service Principal's certificate in the key vault",
+                            "type": "string"
+                        },
+                        "spClientId": {
```

Comment:
Also, can you add to `mlos_bench/README.md` some text on how to get this data? (or add a script, if applicable)

Reply:
I had asked @eujing about creating a second PR for doing the 
1. control plane resources setup
2. RG setup

I think that would be a good place to do the helpers for this, but I do agree that somewhere (e.g., `serivces/remote/azure/README.md` or `config/services/remote/azure/README.md`) a doc that describes the general workflow in some way would be good too.  (it also helps with PR reviews ;)

Reply:
Added a README in `serivces/remote/azure/README.md`!
Will follow up with the second PR about the scripts as mentioned too.

Reply:
Just posted the follow up PR with the scripts! #480